benches/annotation.rs

```diff
@@ -0,0 +1,322 @@
+use std::str::FromStr;
+use std::sync::Arc;
+
+use annonars::common::cli::is_canonical;
+use annonars::common::keys;
+use anyhow::Result;
+use biocommons_bioutils::assemblies::Assembly;
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use noodles_vcf::record::info::field;
+use noodles_vcf::Record;
+use rocksdb::{BoundColumnFamily, DBWithThreadMode, MultiThreaded};
+
+use mehari::annotate::seqvars::csq::{
+    ConfigBuilder as ConsequencePredictorConfigBuilder, ConsequencePredictor, TranscriptSource,
+    VcfVariant,
+};
+use mehari::annotate::seqvars::provider::{
+    ConfigBuilder as MehariProviderConfigBuilder, Provider as MehariProvider,
+};
+use mehari::annotate::seqvars::{
+    annotate_record_auto, annotate_record_clinvar, annotate_record_mt, annotate_record_xy,
+    load_tx_db, CHROM_AUTO, CHROM_MT, CHROM_XY,
+};
+use mehari::pbs::txs::TxSeqDatabase;
+
+const ROCKSDB_SEQVARS_FREQS_PATH: &str = "/mnt/data/mehari/0.21.0/db/grch37/seqvars/freqs";
```

Comment:
I have removed the criterion benchmarks for now; I didn't really use them anyways, since the function+data combinations they were benchmarking were both too expensive and not fine-grained enough. I'd much rather add smaller benches with some custom databases in a different PR.